src/main/java/mate/academy/spring/dao/UserDaoImpl.java

```diff
@@ -0,0 +1,52 @@
+package mate.academy.spring.dao;
+
+import java.util.List;
+import mate.academy.spring.model.User;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class UserDaoImpl implements UserDao {
+    private final SessionFactory sessionFactory;
+
+    @Autowired
```

Comment:
must we Autowired ?

Reply:
Mustn't. But I can. 
And I think I should do it for remember annotation and structure.